code/datums/diseases/scalerot.dm

```diff
@@ -0,0 +1,95 @@
+/datum/disease/scalerot
+	name = "Scale rot"
+	max_stages = 5
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Alcohol based anti-septics or vitamin superdoses."
+	cures = list("vitamin")
+	agent = "Reptilian Necrotizing Dermatitis"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	cure_chance = 10//scalerot, generally speaking, is not easy to cure, and requires lengthy decontamination procedures, and vitamin megadoses are pseudoscience.
+	desc = "A lethal bacteria that affects only scale based skin surfaces. Rots away scales. Commonly carried by various races, and naturally occurs on scaled races experiencing poor hygiene, moist skin, or excess humidty or moisture."
+	severity = DANGEROUS
+	longevity = 650
+
+/datum/disease/scalerot/stage_act()
+	..()
+	if(affected_mob.dna.species.id == "lizard"||affected_mob.dna.species.id == "fly") //Non-scaled species are not affected by the virus and merely carry it.
+		switch(stage)
+			if(1)
+				if(prob(40))
+					if(affected_mob.dna.species.id == "lizard")
+						to_chat(affected_mob << "<span class='notice'>[pick("Your scales feels awfully itchy", "Your tail hurts")]</span>")
+					if(affected_mob.dna.species.id == "fly")
+						to_chat(affected_mob << "<span class='notice'>[pick("Your chitin feels awfully soft", "Your proboscis hurts")]</span>")
+			if(2)
+				if(prob(10))
+					if(affected_mob.dna.species.id == "lizard")
+						to_chat(affected_mob << "<span class='warning'>[pick("You claw at your scales", "Your claws feel strange")]</span>")
+					if(affected_mob.dna.species.id == "fly")
+						to_chat(affected_mob << "<span class='warning'>[pick("Your chitin feels sticky", "Your chitin leaks glucose")]</span>")
+				if(prob(10))
+					affected_mob.adjustStaminaLoss(15)
+			if(3)
+				if(prob(10))
+					if(affected_mob.dna.species.id == "lizard")
+						to_chat(affected_mob.say("hs?"))
+						to_chat(affected_mob << "<span class='danger'>You painfully let out a hiss.</span>")
+					if(affected_mob.dna.species.id == "fly")
+						to_chat(affected_mob.say("buz?"))
+						to_chat(affected_mob << "<span class='danger'>You make a pathetic attempt to buzz, painfully.</span>")
+				if(prob(70))
+					affected_mob.adjustBruteLoss(2)
+				if(prob(10))
+					affected_mob.adjustStaminaLoss(20)
```

Comment:
Massive stamina damage! Fun and engaging gameplay! Let's make a bunch of players fucking squirm and writhe on the floor as they slowly die over five minutes. Fun and ENGAGING gameplay. /sarcasm.